src/sparseml/pytorch/sparsification/quantization/quantize.py

```diff
@@ -308,3 +314,40 @@ def _get_qat_module_mappings() -> Dict[Module, Module]:
         return mappings.get_qat_module_mappings()
     # latest
     return mappings.get_default_qat_module_mappings()
+
+
+def _validate_set_module_schemes(
+    model: Module,
+    scheme_overrides: Optional[Dict[str, QuantizationScheme]] = None,
+    ignore: Optional[List[str]] = None,
+):
+    def _get_unmatched_types_or_names(types_or_names):
+        unmatched = []
+        for type_or_name in types_or_names:
+            matched = False
+            for submodule_name, submodule in model.named_modules():
+                if submodule_name.startswith(type_or_name) or (
+                    submodule.__class__.__name__ == type_or_name
+                ):
+                    matched = True
+                    break
```

Comment:
optional nit: use `not any(...)` here to make it more clear, unsure if it would help.

also think in this case shorter variable names would reduce noise of this snippet